src/redempt/redlib/region/CuboidRegion.java

```diff
@@ -317,6 +318,19 @@ public CuboidRegion getFace(BlockFace face) {
 		region.expand(face.getOppositeFace(), -region.measure(face) + 1);
 		return region;
 	}
+
+	/**
+	 * Gets a random block from within the cuboid region
+	 * @return Random bukkit block from within the region
+	 */
+	public Block getRandomBlock() {
+		Random random = new Random();
```

Comment:
Use ThreadLocalRandom rather than instantiating a new Random instance each time. Better yet, make a second method which takes a Random and uses it to generate the numbers, and a second method which takes no parameters and just passes the ThreadLocalRandom to the variant that takes the Random.